src/Chainweb/Chainweb.hs

```diff
@@ -164,13 +169,34 @@ instance FromJSON (TransactionIndexConfig -> TransactionIndexConfig) where
 pTransactionIndexConfig :: MParser TransactionIndexConfig
 pTransactionIndexConfig = pure id
 
+-- -------------------------------------------------------------------------- --
+-- ReintroTxsConfig
+
+data ReintroTxsConfig = ReintroTxsConfig
+    deriving (Show, Eq, Generic)
```

Comment:
Is there a chance that more fields are added to this constructor? Otherwise a simple boolean flag in `ChainwebConfiguration` would have been sufficient.

Introducing a dedicated type is more future proof / forward compatible if more settings are added.